_15thnight/api/account.py

```diff
@@ -74,16 +74,17 @@ def forgot_password():
                 user.reset_created_at < datetime.utcnow() - reset_token_life):
             user.generate_reset_token()
         user.save()
+
         link = '%sreset-password/%s/%s' % (
-            url_for('index', _external=True), user.email, user.reset_token
-        )
+            url_for('index', _external=True), user.email, user.reset_token)
```

Comment:
Let's keep this formatting the way it was originally. We are doing it below for `Message`. Consistency is important.